### DISCUSSION TITLE
Upgrades for nonprofits

### DISCUSSION THREAD

**Discussion by NewEyes-Evoucher** [community/community#37524] [Other Feature Feedback, Questions, & Ideas]
Looking to upgrade our nonprofit account to Team account (GitHub states it ofers free upgrade to nonprofits)> I have emailed Billing twicw and no answer. Any recos on hwho to contact/how to get upgrade?

**Comment by Chernington** [comment]
GitHub has a page for complying 501(c)(3) non-profit organizations. You can apply [here](https://support.github.com/contact/nonprofit) for access to premium features.
